access-mgmt/README.adoc

```diff
@@ -0,0 +1,473 @@
+
+.Prerequisites
+* You're logged in to the {product} web console with administrator access rights. If you're managing only service accounts that you created for a Kafka instance that you created, you don't need administrator rights.
+* You have access to one or more running Kafka instances in {product}.
+* The user account or service account that you're setting permissions for has been created in the organization.
+
+.Procedure
+. In the {product} web console, go to *Streams for Apache Kafka* > *Kafka Instances* and click the name of the Kafka instance that you want to set permissions for.
+. Click the *Permissions* tab to view the current ACL permissions for this instance.
++
+--
+By default, new Kafka instances contain the following permissions. These permissions allow all accounts in the organization to view topics in the instance, but not to produce or consume messages.
+
+NOTE: The wildcard character `*` denotes any occurrences of the specified resource. For example, `Topic` = `is *` means any topics in a Kafka instance.
+
+.Default ACL permissions for new Kafka instances
+[cols="25%,25%,25%,25%"]
+|===
+h|Account
+h|Resource
+h|Access type
+h|Operation
+
+|`All accounts`
+|`Topic is *`
+
+(Any topics)
+|`Allow`
+|`Describe`
+
+|`All accounts`
+|`Consumer group is *`
+
+(Any consumer groups)
+|`Allow`
+|`Describe`
+
+|`All accounts`
+|`Transactional ID is *`
+
+(Any producers)
+|`Allow`
+|`Describe`
+|===
+--
+. Use the *Permissions* page to set permissions for a new account, add permissions to an existing account, or delete account permissions in this instance.
++
+To set permissions for a new account in this instance, follow these steps:
+
+.. Click *Manage access*.
+.. In the *Account* drop-down menu, select the new user account or service account that you want to set permissions for. You can also select *All accounts* to set permissions for all user accounts and service accounts in the organization.
+.. Click *Next*.
+.. Under *Review existing permissions*, verify the current permission settings for this instance. You can delete existing permissions now if needed, or you can wait to delete existing permissions later from the main *Permissions* page.
++
+If you previously selected a specific account, you can delete only permission entries that apply to individual accounts. If you previously selected *All accounts*, you can delete only permission entries that apply to all accounts.
+
+.. Under *Assign Permissions*, set the permissions for the specified account or all accounts for a resource type, such as a topic, and click *Add* to continue adding permissions for other resources as needed.
++
+--
+For example, when you create a new service account, add the following permissions in the Kafka instance that you want the account to access. These permissions enable applications associated with the service account to produce and consume messages in any topics in the instance, using any consumer group and any producer.
```

Comment:
This reads like I "HAVE" to do that for each service account, i.e. that I have to add those exact permissions. However, I could also have a service account that is only allowed to consume from a given instance, but not produce to it. So I'd rather say something like:

"For example, when you create a new service account, you can add the the following permissions in the Kafka instance that you want the account to access."

Reply:
Sure, I just thought we wanted to be more prescriptive with users here, to tell them to just create these permissions for new service accounts. But you're right, it's one thing to be prescriptive in the getting started like we've done to achieve a specific tutorial outcome, but here it's just an option. So I've tweaked as you said, with "you can" and "in this example" to emphasize that it's just one option.